runtime/libpgmath/tools/tmp-mth_statsdefs_generic.ps1

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
```

Comment:
Use new license and remove copyright. Let me know if there is a particular reason to use the old license and copyright.